lib/output.js

```diff
@@ -67,7 +67,8 @@ function OutputStream(options) {
         screw_ie8        : false,
         preamble         : null,
         quote_style      : 0,
-        keep_quoted_props: false
+        keep_quoted_props: false,
+        ecma5            : true
```

Comment:
Also can you use trailing comma for last object property to make future diffs smaller and easier to read?